modules/keda/variables.tf

```diff
@@ -14,3 +15,15 @@ variable "values" {
   type        = map(any)
   default     = {}
 }
+
+variable "namespace" {
+  description = "The namespace to install in, defaults to tenant namespace. NOTE: only one KEDA install per cluster is supported!"
+  type        = string
+  default     = null
+}
+
+variable "sets" {
```

Comment:
Won't this still have the issue with arbitrary types? 
Like what if I set a boolean and a string? 

Can we simply have a variable that asks for a string of values? Then add that string to the list of values.

Reply:
converted it to a map of string since set takes strings exclusively.